data/src/main/java/com/nemethedgar/data/features/news/local/NewsLocalDataStore.java

```diff
@@ -0,0 +1,47 @@
+package com.nemethedgar.data.features.news.local;
+
+import com.nemethedgar.data.features.news.mapper.NewsArticlesToEntity;
+import com.nemethedgar.data.features.news.mapper.NewsEntityToArticle;
+import com.nemethedgar.data.features.news.model.Article;
+
+import java.util.List;
+
+import io.reactivex.Completable;
+import io.reactivex.Single;
+import io.reactivex.schedulers.Schedulers;
+
+public class NewsLocalDataStore {
+
+    private static NewsDao dao;
```

Comment:
let's make the dao final but static, why is it static? We should avoid static members, as are less testable, prone to memory leaks

Reply:
done